language/tools/move-cli/src/utils/movey_credential.rs

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) The Diem Core Contributors
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, Context, Result};
+use move_command_line_common::movey_constants::MOVEY_URL;
+use std::fs;
+use toml_edit::easy::Value;
+
+pub const MOVEY_CREDENTIAL_PATH: &str = "/movey_credential.toml";
```

Comment:
This constant is also defined in [movey_login.rs](https://github.com/move-language/move/blob/514cc0003a66f352f0259239ed688441e06da74c/language/tools/move-cli/src/base/movey_login.rs#L10).